modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -102,6 +102,18 @@ MS2-Status-Code:: Specifies the status code or codes that the mocking service ca
 If you use this header together with MS2-Error-Rate, you must specify both a non-error status code and an error status code. Separate the two status codes with a comma. For example, you might want the mocking service to return a 201 message, if there is no error, or a 401 status code, if there is an error; in this case, you would specify `201,401` for the value.
 ====
 
+MS2-Status-Code-Selection:: By default, when the mocking service responds to a request, it chooses the minimum status code that is defined for the corresponding method. For example, if you make a POST request to an endpoint that defines responses for status codes 201, 400, 409, and 500 in this order, by default the mocking service sends the response message that is defined for the 201 status code. With the MS2-Status-Code-Selection header, you can change this default behavior or the mocking service.
++
+The possible values are:
++
+`FIRST`:: The mocking service sorts the list of status codes in ascending numeric order and returns the first. For example, if the specification defines the status codes for a POST method in the order 500, 409, 400, 200, the mocking service returns the response for the 200 status code.
++
+You can combine the use of this value of MS2-Status-Code-Selection with the use of MS2-Error-Level. The mocking service sorts in ascending numeric order the status codes that are in the range of error status codes specified by MS2-Error-Level, and then returns the first error status code from that range.
++
+`RANDOM`:: The mocking service sorts the list of status codes in ascending numeric order and selects one randomly to return. For example, if the specification defines the status codes for a POST method in the order 500, 409, 400, 200, the mocking service sorts this list and then returns the response for the any one of the status codes.
++
+You can combine the use of this value of MS2-Status-Code-Selection with the use of MS2-Error-Level. The mocking service sorts in ascending numeric order the status codes that are in the range of error status codes specified by MS2-Error-Level, and then returns a randomly chosen error status code from that range.
```

Comment:
This paragraph is similar to the one stated for the `FIRST` value. 

I would remove one of them and I would change this ```and then returns a randomly chosen error status code from that range.``` for something like this ```and then returns an error status code from that range, chosen following the strategy specified by MS2-Status-Code-Selection```

Reply:
Done.